core/globals.h

```diff
@@ -293,6 +293,8 @@ typedef struct _dr_stats_t {
      *  or capacity reasons or thread-private caches.
      */
     uint64 basic_block_count;
+    /** Total number of threads ever under DR control. */
```

Comment:
The description is ambiguous: please clarify that this is the peak simultaneous number.  (Is that what you wanted to export, or the accumulated total?)

Reply:
I wanted the accumulated total. Is "num_threads_created" the appropriate statistic? Will it count app threads which were active when DR takes over?

Reply:
I poked around some more and it looks like num_threads_created is more appropriate for our internal use case so I've added that as well.